src/components/ChatBubble.tsx

```diff
@@ -0,0 +1,58 @@
+import styled from "styled-components";
+import "../style/color.css";
+import { Body2 ,Caption2} from "../style/font";
+interface ChatBubbleProps{
+    isMyChat:boolean;
+    message :string;
+    chatDate : string;
+}
+function ChatBubble(props: ChatBubbleProps){
+    const date = new Date(props.chatDate);
+    const hours = date.getHours();
+    const minutes = date.getMinutes();
+    const isAM = hours < 12 ? true : false;
+    const hours12 = hours < 13 ? hours : hours-12;
```

Comment:
date를 새로운 형식으로 변환하는 기능은 해당 컴포넌트에 의존적이지도 않고 추후에 다른 곳에서도 사용될 수 있으니 다른 파일에서 함수로 작성해보는 것도 좋을 것 같아요~~
```suggestion
    const convertDate = (originalDate: string) => {
        const date = new Date(originalDate);
        const hours = date.getHours();
        const minutes = date.getMinutes();
        const isAM = hours < 12 ? true : false;
        const hours12 = hours < 13 ? hours : hours-12;
    }

```

Reply:
재사용 가능한 함수나 컴포넌트에 대해서 잘 고민하면서 구현해보겠습니다!